showcase/grpc-gapic-showcase-v1beta1/pom.xml

```diff
@@ -34,11 +34,10 @@
                 </goals>
                 <phase>test</phase>
                 <configuration>
-                  <executable>bazel</executable>
+                  <executable>bash</executable>
                   <arguments>
-                    <argument>run</argument>
-                    <argument>--experimental_proto_descriptor_sets_include_source_info</argument>
-                    <argument>//showcase:verify_grpc</argument>
+                    <argument>../scripts/verify.sh</argument>
```

Comment:
Do we have to repeat this plugin configuration in every submodule? Can we just re-generate all of them from the top-level module?

Reply:
Makes sense. I moved the profile to the parent pom so golden tests and source code update is performed in a single place